### DISCUSSION TITLE
concat

### DISCUSSION THREAD

**Discussion by brunocalado** [EddieDover/Theater-of-the-Mind#64] [Ideas]
Hi 
Can you you add a concat funcion?
I wanted the "D 6" to be "D6", but I can't find a way to do this without a concat function.
![image](https://github.com/EddieDover/Theater-of-the-Mind/assets/4999783/a69d8fd4-1b46-4b84-bded-7f8aa7d85d12)

Thank you.

**Comment by EddieDover** [comment]
This will be possible after #66 is merged. It adds new tags for spacing, including single spaces {s}, multiple spaces {s#}, and space negation {s0}.
